### PR TITLE
CORTX-29731: system_health even-id getting floating value

### PR DESCRIPTION
**Pull request by gauri-bhosale**
Signed-off-by: gauri-bhosale <gauri.bhosale@seagate.com>

# Problem Statement
-  https://jts.seagate.com/browse/CORTX-29731

# Design
- event_id should contain timestamp but not should contain floating value 
- so need to convert into integer before string conversion

# Coding 
-  Coding conventions are followed and code is consistent [Y/N]:  Y
-  Confirm All CODACY errors are resolved [Y/N]: Y

# Testing
- Are test cases updated along with code changes due to Enhancements/Bugs [Y/N]: Y
- Confirm that new test cases are added to regression and sanity plan files and relevant feature plan files [Y/N]: Y
- Confirm that Test Cases are added for new features added [Y/N]:  Y 
- Confirm Testing was performed with installed RPM/K8s deployment [Y/N]:  Y

# Review Checklist 
####  Before posting the PR please ensure:
- [X] PR is self reviewed
- [X] Jira is updated
- [X] Check if the description is clear and explained. 
- [X] Check Acceptance Criterion is defined. 
- [X] All the tests performed should be mentioned before Resolving a JIRA. 
- [X] Verification needs to be done before marked as Closed/Verified.

# Documentation
- [X] Changes done to WIKI / Confluence page

Testing result :
sh-4.4# pytest test_health_event.py
======================================= test session starts ========================================
platform linux -- Python 3.6.8, pytest-7.0.1, pluggy-1.0.0
rootdir: /etc/cortx
collected 3 items

test_health_event.py ...                                                                     [100%]

======================================== 3 passed in 0.02s =========================================
sh-4.4#

Sample local test :
[root@ssc-vm-g3-rhev4-2813 event_framework1]# python3 health.py
if returnkeysss
{"header": {"version": "1.0", "timestamp": "1651219774.4916775", "event_id": "16512197748aef6b536ea641f39371519fe45fabbe"}, "payload": {"source": "", "cluster_id": "", "site_id": "", "rack_id": "", "storageset_id": "", "node_id": "", "resource_type": "", "resource_status": "", "specific_info": ""}}
if returnkeysss
if returnkeysss
{"header": {"version": "1.0", "timestamp": "1651219774.494863", "event_id": "1651219774fee9e4c295114e5b8a16dcd59937eff9"}, "payload": {"source": "s1", "cluster_id": "1234", "site_id": "1", "rack_id": "1", "storageset_id": "1", "node_id": "3", "resource_type": "node", "resource_status": "ok", "specific_info": {"a": "a1", "b": true}}}
[root@ssc-vm-g3-rhev4-2813 event_framework1]#

### CODE REVIEW

**Review by shaileshsvc — APPROVED** [review]

**Review by Madhura-08 — APPROVED** [review]

File: py-utils/src/utils/event_framework/event.py [hunk 1]
```diff
@@ -44,11 +44,11 @@ def __init__(self, payload: KvPayload = KvPayload()):
...
         super().__init__()
         # Set Header
-        _now = str(time.time())
+        _now = time.time()
         _uuid = str(uuid.uuid4().hex)
         super().set(f'header>{EventAttr.VERSION}', VERSION)
-        super().set(f'header>{EventAttr.TIMESTAMP}', _now)
-        super().set(f'header>{EventAttr.EVENT_ID}', _now + _uuid)
+        super().set(f'header>{EventAttr.TIMESTAMP}', str(_now))
+        super().set(f'header>{EventAttr.EVENT_ID}', str(int(_now)) + _uuid)
```
Comment (gargpalak048) [inline py-utils/src/utils/event_framework/event.py:51]: so, now timestamp can be a floating value
but event-id will be only integer value.
correct @gauri-bhosale

**Review by gargpalak048 — APPROVED** [review]
approved with one small query

File: py-utils/src/utils/event_framework/event.py (hunk 1 above)
Comment (gauri-bhosale) [reply py-utils/src/utils/event_framework/event.py:51]: yes, As we have not specific reason to convert int timestamp as its generic code may other component need timestamp in float so keeping that.

**Review by ArchanaLimaye — APPROVED** [review]

**Review by ujjwalpl — APPROVED** [review]

**Review by cdeshmukh — APPROVED** [review]

**Comment by cdeshmukh** [comment]
Can merge once 1N deployment is complete

**Pull request merged**